vllm/engine/arg_utils.py

```diff
@@ -1095,10 +1096,11 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             "If enabled, the model will be able to generate reasoning content."
         )
 
+        reasoning_parsers = ReasoningParserManager.reasoning_parsers.keys()
         parser.add_argument(
             "--reasoning-parser",
             type=str,
-            choices=["deepseek_r1"],
+            choices=[*reasoning_parsers],
```

Comment:
we can just do

```suggestion
            choices=list(ReasoningParserManager.reasoning_parsers)
```
?